doc/deployment/deploy_custom.md

```diff
@@ -84,203 +43,219 @@ pachctl deploy custom --persistent-disk <persistent disk backend> --object-store
+The following flags set attributes which are passed on to Kubernetes directly through the produced manifest.
+Do not modify the default values of these flags for production deployments without consulting with Pachyderm support.
+- `--etcd-cpu-request`: The number of CPU cores that Kubernetes allocates to `etcd`. Fractions are allowed. 
+- `--etcd-memory-request`: The amount of memory that Kubernetes allocates to `etcd`. The SI suffixes are accepted as possible values.
+- `--no-guaranteed`: Turn off QoS for `etcd` and `pachd`. Do not use this flag in production environments.
+- `--pachd-cpu-request`: The number of CPU cores that Kubernetes allocates to `pachd`. Fractions are allowed. 
+- `--pachd-memory-request`: The amount of memory that Kubernetes allocates to `pachd`. This flag accepts the SI suffixes.
+- `shards`: The maximum number of `pachd` nodes allowed in the cluster. Increasing this number from the default value of 16 might result in degraded performance.
+
 
 #### Enterprise Edition features
 
-- `--dash-image`: the docker image for the Pachyderm Enterprise Edition dashboard
-- `--image-pull-secret`: the name of a Kubernetes secret Pachyderm will need to pull from a private docker registry
-- `--no-dashboard`: don't create a manifest for deploying the Enterprise Edition dashboard
-- `--registry`: the registry for docker images
-- `--tls`:  string of the form `"<cert path>,<key path>"` of the signed TLS certificate used for encrypting pachd communications
+- `--dash-image`: The Docker image for the Pachyderm Enterprise Edition dashboard.
+- `--image-pull-secret`: The name of a Kubernetes secret that Pachyderm uses to pull from a private Docker registry.
+- `--no-dashboard`: Skip the creation of a manifest for the Enterprise Edition dashboard.
+- `--registry`: The registry for Docker images.
+- `--tls`:  A string in the `"<cert path>,<key path>"` format with the signed TLS certificate that is used for encrypting `pachd` communications.
+
 
 #### Output formats
 
-- `--dry-run`: don't actually deploy to Kubernetes, just send the manifest to standard output
-- `-o` or `--output`: choose from json (the default) or yaml
+- `--dry-run`: Create a manifest and send it to standard output, but do not deploy to Kubernetes.
+- `-o` or `--output`: An output format. You can choose from JSON (default) or YAML.
 
 #### Logging
-- `log-level`: sets the verbosity level of `pachd`, from most verbose to least the settings are `debug`, `info`, and `error`.
-- `-v` or `--verbose`: controls the chattiness of the `pachctl` invocation itself.
+- `log-level`: The `pachd` verbosity level, from most verbose to least. You can set this parameter to `debug`, `info`, or `error`.
+- `-v` or `--verbose`: Controls the verbosity of the `pachctl` invocation.
 
 #### Complete example invocation
 Since we're deploying to a code infrastructure that works with YAML files,
```

Comment:
this whole first paragraph is awkwardly worded. I get that we're just trying to show an end-to-end example. So just say so.  Also, the phrase "code infrastructure" get used here and elsewhere and it sounds awkward. I understand the tagline "infra as code", but its not a noun like you use keep using here (and elsewhere, please fix those too). 

"Here's a complete deploy command example of a custom deployment. In this example, the infrastructure expects a YAML manifest and uses `local-roles` instead of cluster-wide roles so there are optional flag sets for those constraints. " <-- Am i missing something or is this a much simpler phrasing to get the same point across?

```
pachctl deploy custom ..... --dry-run
```